### ISSUE TITLE
Issue with "OLD SSL Reference": (almost) all links are dead anchor links ?!

### ISSUE THREAD

**Issue by CapitainFlam** [mdn/content#10904]
MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/SSL_functions/OLD_SSL_Reference

#### What information was incorrect, unhelpful, or incomplete?
Almost all links are written as this link :
```
<h4 id="Chapter_1_Overview_of_an_SSL_Application"><a href="sslintro.html#1028068">Chapter 1  Overview of an SSL
    Application </a></h4>
```
where links are NOT written as  `href="sslintro"` but as `<a href="sslintro.html#1028068">`. 
This means that links are DEAD.

In the destination page, like [sslintro](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/SSL_functions/sslintro), we can find in the code the anchors as we spoke : 
```
<h2 id="Chapter_1_Overview_of_an_SSL_Application">Chapter 1<br>
  <a id="1028068"> Overview of an SSL Application</a></h2>
<p><a id="1028153"> SSL and related APIs allow compliant applications to configure sockets for authenticated, tamper-proof, and encrypted communications. This chapter introduces some of the basic SSL functions. </a><a href="gtstd.html#1005439">Chapter 2, "Getting Started With SSL"</a> illustrates their use in sample client and server applications.</p>
<p><a id="1028234"> An SSL application typically includes five parts:</a></p>
<p><a id="1028157"> </a><a href="#1027662">Initialization</a><br>
  <a href="#1027742">Configuration</a><br>
  <a href="#1027816">Communication</a><br>
  <a href="#1027820">Functions Used by Callbacks</a><br>
  <a href="#1030535">Cleanup</a></p>
```

#### Specific section or headline?
All anchored links.

#### What did you expect to see?
Working links. The hard way would be to remove all anchor, but it would certainly be to violent as a method.

#### Did you test this? If so, how?
if we take the first exemple again, the link goes to : 
[https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/SSL_functions/sslintro.html#1028068](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/SSL_functions/sslintro.html#1028068)
and shows : 
```
Page not found

Sorry, the page /en-US/docs/Mozilla/Projects/NSS/SSL_functions/sslintro.html could not be found.
```

if you correct the HTTP link to : 
[https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/SSL_functions/sslintro](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/SSL_functions/sslintro)
it shows the correct page. 

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/mozilla/projects/nss/ssl_functions/old_ssl_reference`
* MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/SSL_functions/OLD_SSL_Reference
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/mozilla/projects/nss/ssl_functions/old_ssl_reference/index.html
* Last commit: https://github.com/mdn/content/commit/7ce080a8d79c3ba23048cb7e28655818d6e49efb
* Document last modified: 2021-08-23T21:16:18.000Z

</details>

**Comment by hamishwillee** [comment]
Closing - this content is all supposed to be off MDN already. See https://github.com/mdn/content/issues/10903 and https://github.com/mdn/content/pull/8215

Thanks for posting.

**Comment by dnldelRey** [comment]
Idk what youre talking bout, but hey its nice to see somebody on my inbox  Enviado desde Correo para Windows De: Hamish WilleeEnviado: jueves, 2 de diciembre de 2021 05:36 p. m.Para: mdn/contentCC: SubscribedAsunto: Re: [mdn/content] Issue with "OLD SSL Reference": (almost) all links are dead anchor links ?! (Issue #10904) Closed #10904.—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
